app/frontend/src/components/LabelAndText.tsx

```diff
@@ -13,6 +14,9 @@ const useStyles = makeStyles((theme) => ({
     display: "flex",
     marginTop: theme.spacing(0.5),
   },
+  flexItem: {
+    flex: "1 1 0px",
```

Comment:
```suggestion
    flex: "1 1 50%",
```
I think it may be worth setting the 50% width via `flex-basis` than using `width` if we are using these flex properties 🤔 
That means you should be able to remove the `width: 50%` on line 11

Reply:
Yeah that makes sense @darrenvong .
Pushed the changes.